src/stdlib_strings_format_string.fypp

```diff
@@ -0,0 +1,37 @@
+#:include "common.fypp"
+#:set RIL_KINDS_TYPES = REAL_KINDS_TYPES + INT_KINDS_TYPES + LOG_KINDS_TYPES
+submodule (stdlib_strings) stdlib_strings_format_string
+
+    implicit none
+    integer, parameter :: buffer_len = 512
```

Comment:
The `buffer_len` should bre mentioned in the specs (note that I may have miss it).